packages/frontend/src/components/message/Message.tsx

```diff
@@ -792,15 +792,18 @@ export const Quote = ({
               )}
             </>
           )}
-          <div className='quoted-text'>
-            <MessageBody
-              text={
-                quote.text.slice(0, 3000 /* limit quoted message size */) || ''
-              }
-              disableJumbomoji
-              nonInteractiveContent
-            />
-          </div>
+          {hasMessage && quote.text && (
```

Comment:
Doesn't this hide the quote when `quote.kind === 'JustText'`? Because `hasMessage` is defined as `quote.kind === 'WithMessage'`.

And I think we need a comment for the `quote.text &&` check, at least link the issue.

Reply:
So now I check only for the text property to exist.

> And I think we need a comment for the quote.text && check, at least link the issue

I think it is self explaining to check for the text property to exist before calling "slice" on it.

Reply:
Well, the thing is that the types say that it always exists.